### ISSUE TITLE
After restarted TB, wrench options button not showing

### ISSUE THREAD

**Issue by Francewhoa** [cleidigh/printing-tools-ng#153]
To make it easier and faster to find and open the **printing-tools-ng**'s configuration. I suggest adding an **Add-On Option** button. 

**Steps:**

1. Using Thunderbird, user navigate to **Tools** menu ---> **Add-ons and Themes** option

1. The number 1 in this mockup shows the suggested location of this **Add-On Option** button. Icon is a wrench.

   ![mockup------Add-on Option---francewhoa---ksnip---2022-12-08---180823---v2---flatten](https://user-images.githubusercontent.com/119942/206588130-82f427ce-50dd-4e9e-ab5c-eb79bc41e97e.png)
   
   For privacy, part of the content of this screenshot above was manually masked. This is normal. Ignore this.

1. This screenshot shows that after the user click on this new **Add-On Option** button, this **PrintingTools NG** windows would open.

   ![screenshot---printingtools NG---window---francewhoa---ksnip---2022-12-08---182814](https://user-images.githubusercontent.com/119942/206588750-6f01928a-00e7-4582-80e9-14d66d98c8b2.png)

   For privacy, part of the content of this screenshot above was manually masked. This is normal. Ignore this.

1. Alternatively, the user could still open this **PrintingTools NG** windows using this already existing **Files** menu ---> **PrintingTools NG** option. Most new users use **Tools** menu ---> **Add-ons and Themes** option though.

**Comment by Francewhoa** [comment]
This screenshot shows the present **Manager Your Extension** window with **printing-tools-ng**. Before adding the new **Add-On Option** button.

![screenshot---before adding---Add-on Option---francewhoa---ksnip---2022-12-08---180823---v2](https://user-images.githubusercontent.com/119942/206590318-85930bba-adab-4514-93cf-90f79c3ddf31.png)

**Comment by Francewhoa** [comment]
Somehow, the **Add-On Option** button is now displayed. Maybe this **Add-On Option** button was already added. But somehow not showing on my Thunderbird 102.5.0

**Steps:** 

1. Thunderbird (TB) v102.5.0, before testing [this new **printing-tools-ng** (PTNG)](https://github.com/cleidigh/printing-tools-ng/issues/152) I clicked on the blue on-off toggle button. To set PTNG v2.5.1 to **off**. The [number 1 in this screenshot](https://user-images.githubusercontent.com/119942/206590318-85930bba-adab-4514-93cf-90f79c3ddf31.png) shows this toggle. I did this before I uninstall it and try the new **printing-tools-ng** version. 

1. Still using 2.5.1, before installing the new version of  PTNG I toggle PTNG back to **on**. Somehow, doing this displayed the **Add-On Option** button. 

1. I'm guessing that this **Add-On Option** button was already added in PTNG 2.5.1. But somehow not showing on my TB 102.5.0. Until I did the magic-toggle-thingy above. I'm guessing that the toggle thing above somehow either refresh PTNG button, or clear some cache, or overrode a old version of PTNG, or maybe this was just a glitch in my TB.

Anyhow, I'm closing this ticket as **Add-On Option** is obviously already implemented

In the future, if someone else notices the same "glitch", you're welcome to re-open this ticket

**Comment by RRoenn** [comment]
@Francewhoa 
do you really have a black window?

> 3. This screenshot shows that after the user click on this new Add-On Option button, this PrintingTools NG windows would open.
> ![screenshot---before adding---Add-on Option---francewhoa---ksnip---2022-12-08---180823---v2](https://user-images.githubusercontent.com/119942/206588750-6f01928a-00e7-4582-80e9-14d66d98c8b2.png)

After installation the Add-On you have to restart TB, otherwise you will have strange experiences ...

**Comment by tidebre67** [comment]
@cleidigh , do not have this problem, no blank page/s  with current TB 102.5.1.

**Comment by cleidigh** [comment]
@tidebre67 
I think what I have heard multiple people comment is that frequently they have to restart TB to get the wrench icon for the options.
It has to do with the startup sequence.
@cleidigh

**Comment by Francewhoa** [comment]
Good morning @RRoenn :) Thanks for your comment. My replies below.

--------------------------

>do you really have a black window?

No

For privacy, part of the content of this screenshot above was manually masked. This is normal. Ignore this.

By "masked" I mean the content is blurred out into an almost black

That challenge is that the wrench button is not displayed by default. The masked screenshot is not the challenge.

--------------------------


>After installation the Add-On you have to restart TB, otherwise you will have strange experiences ...

This did not work in my case

For the past months I restarted TB many times. But this failed. The wrench button did not show. 

The only thing that worked for me was those two steps:

1. Restart TB

2. Toggle the on-off button. Detailed steps in my comment [#1343572598](https://github.com/cleidigh/printing-tools-ng/issues/153#issuecomment-1343572598) above

**Comment by Francewhoa** [comment]
Good morning @cleidigh and @RRoenn :)

>After installation the Add-On you have to restart TB, otherwise you will have strange experiences ...

>I think what I have heard multiple people comment is that frequently they have to restart TB to get the wrench icon for the options.

Restarting TB did not work in my case

For the past months I restarted TB many times. But this failed. As the wrench button still did not show.

Still in my case, the only thing that resolved this challenge was those two steps:

1. Restart TB

2. Toggle [this blue on-off button](https://user-images.githubusercontent.com/119942/206590318-85930bba-adab-4514-93cf-90f79c3ddf31.png). Detailed steps in my comment [#1343572598](https://github.com/cleidigh/printing-tools-ng/issues/153#issuecomment-1343572598) above

-----------------

@All :) I'm assuming that, somehow, I'm the only user who needed to do those 2 steps. Not just the 1st step.

In the future, if someone else notices the same "glitch" about the wrench button not showing even after a restart, you're welcome to re-open this ticket

I updated the title of this ticket to clarify the two steps above
